Userland/Libraries/LibWeb/HTML/NavigationParams.h

```diff
@@ -62,6 +62,9 @@ struct NavigationParams {
     // FIXME: an algorithm expecting a response
     void* process_response_end_of_body { nullptr };
 
+    // null or a fetch controller
+    JS::GCPtr<Fetch::Infrastructure::FetchController> fetch_controller { nullptr };
```

Comment:
So this GCPtr is safe because `NavigationParams` is always on the stack? That's probably okay but it does feel slightly rickety.